respokeSDK/src/main/java/com/digium/respokesdk/RespokeClient.java

```diff
@@ -511,6 +531,170 @@ public RespokeGroup getGroup(String groupIDToFind) {
         return group;
     }
 
+    public void getGroupHistories(ArrayList<String> groupIds,
```

Comment:
Since this is a public API, is it worth documenting that this only pulls the first message back for each group?